code/go/0chain.net/chaincore/block/block_eventdb.go

```diff
@@ -29,7 +30,9 @@ func blockToBlockEvent(block *Block) *event.Block {
 		StateChangesCount:     block.StateChangesCount,
 		RunningTxnCount:       fmt.Sprintf("%d", block.RunningTxnCount),
 		RoundTimeoutCount:     block.RoundTimeoutCount,
-		CreatedAt:             block.CreationDateField.ToTime(),
+		UpdatableModel:        model.UpdatableModel{
```

Comment:
createdAt will be set automatically https://gorm.io/docs/models.html#Creating-x2F-Updating-Time-x2F-Unix-Milli-x2F-Nano-Seconds-Tracking

Reply:
I was trying to do that before but VS code was giving me error.